docs/introduction/installation.md

```diff
@@ -43,6 +43,12 @@ Docker will mount the configuration files and icons to your host machine. Please
 
 :::
 
+:::caution
+
+Make sure all your volumes path are pointing at different folders or the mounting will fail.
```

Comment:
Can you give an example here? Most users should understand this already, but a few won't, because Homarr is the first app that they ever install with Docker.